src/components/RightPane/CoursePane/SchoolDeptCard.tsx

```diff
@@ -34,11 +36,18 @@ const styles = (theme) => ({
     },
 });
 
-class SchoolDeptCard extends PureComponent {
+interface SchoolDeptCardProps {
+    classes: ClassNameMap;
+    comment: string;
+    name: string;
+    type: string;
+}
+
+class SchoolDeptCard extends PureComponent<SchoolDeptCardProps> {
     state = { commentsOpen: false };
 
     render() {
-        const html = { __html: [this.props.comment] };
+        const html = { __html: this.props.comment };
```

Comment:
Why remove the list here? Not saying we shouldn't, but this is a change that isn't just adding types which is why I'm curious.

Reply:
The `dangerouslySetInnerHTML` prop takes an object of type `{__html: string}`. I have no idea how it worked when passed an object with a `string[]` instead (probably more JavaScript spaghetti).